automation/WinFormsAutomationApp/GlobalSuppressions.cs

```diff
@@ -0,0 +1,10 @@
+﻿
+// This file is used by Code Analysis to maintain SuppressMessage 
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given 
+// a specific target and scoped to a namespace, type, member, etc.
+
+[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("AsyncUsage.CSharp.Reliability", "AvoidAsyncVoid:Avoid async void", Justification = "<Pending>", Scope = "member", Target = "~M:WinFormsAutomationApp.MainForm.RequestGo_Click(System.Object,System.EventArgs)")]
```

Comment:
><Pending> [](start = 144, length = 9)

event handlers of a Winforms event which requires a void signature. Alternatively you can create a method which calls the methods needed.

Reply:
@henrik-me @jmprieur 
In regards to the suppressed errors, those are temporary just to get the project to build. I wanted to have a discussion on this. Currently, this async analyzer will show an error for any async function that returns void. However, there are functions in our code that currently require void to be returned (e.g. Event Handlers like Henrik mentioned). do we want this rule to be enforced or should we turn it off?